### PR TITLE
Remove more BC layers

### PR DESCRIPTION
**Pull request by leofeyer**

### CODE REVIEW

File: core-bundle/src/Resources/contao/classes/BackendUser.php [hunk 1]
```diff
@@ -529,7 +529,6 @@ public function navigation($blnShowAll=false)
 				$arrModules[$strGroupName]['label'] = ($label = \is_array($GLOBALS['TL_LANG']['MOD'][$strGroupName] ?? null) ? ($GLOBALS['TL_LANG']['MOD'][$strGroupName][0] ?? null) : ($GLOBALS['TL_LANG']['MOD'][$strGroupName] ?? null)) ? $label : $strGroupName;
 				$arrModules[$strGroupName]['href'] = $router->generate('contao_backend', array('do'=>Input::get('do'), 'mtg'=>$strGroupName, 'ref'=>$strRefererId));
 				$arrModules[$strGroupName]['ajaxUrl'] = $router->generate('contao_backend');
-				$arrModules[$strGroupName]['icon'] = 'modPlus.gif'; // backwards compatibility with e.g. EasyThemes
```
Comment (leofeyer) [inline core-bundle/src/Resources/contao/classes/BackendUser.php:532]: @Toflar Do you still need this?

Comment (Toflar) [reply core-bundle/src/Resources/contao/classes/BackendUser.php:532]: No idea but I guess it's to be deleted anyway :P

Comment (leofeyer) [reply core-bundle/src/Resources/contao/classes/BackendUser.php:532]: No, we can keep it if you want. That‘s why I asked. 😄

Comment (Toflar) [reply core-bundle/src/Resources/contao/classes/BackendUser.php:532]: Just drop it ;)

File: manager-bundle/src/Composer/ScriptHandler.php [hunk 2]
```diff
@@ -48,11 +48,6 @@ public static function initializeApplication(Event $event): void
         $command[0] = Path::join(__DIR__.'/../../bin', $command[0]);
         array_unshift($command, $phpPath);
 
-        // Backwards compatibility with symfony/process <3.3 (see #1964)
-        if (method_exists(Process::class, 'setCommandline')) {
-            $command = implode(' ', array_map('escapeshellarg', $command));
-        }
-
```
Comment (m-vo) [inline manager-bundle/src/Composer/ScriptHandler.php:55]: IMHO the whole `ScriptHandler` file can be dropped (see L32).

**Review by m-vo — CHANGES_REQUESTED** [review]

File: manager-bundle/src/Composer/ScriptHandler.php (hunk 2 above)
Comment (leofeyer) [reply manager-bundle/src/Composer/ScriptHandler.php:55]: You are right. Removed in 2a2edd24cdf6add386adf9038ff6a480ab768b1c.

**Review by m-vo — APPROVED** [review]

**Pull request merged**